src/libraries/AccountAccessParser.sol

```diff
@@ -135,8 +135,9 @@ library AccountAccessParser {
 
     bytes32 internal constant LIVENESS_GUARD_LAST_LIVE_SLOT = bytes32(uint256(0));
 
-
     bytes32 internal constant ANCHOR_STATE_REGISTRY_RETIREMENT_TIMESTAMP_SLOT = bytes32(uint256(6));
+    bytes32 internal constant ANCHOR_STATE_REGISTRY_PROPOSAL_ROOT_SLOT = bytes32(uint256(3));
+    bytes32 internal constant ANCHOR_STATE_REGISTRY_PROPOSAL_L2_SEQUENCE_NUMBER_SLOT = bytes32(uint256(4));
```

Comment:
Are we able to able to avoid harcoding any of these and use forge-std's `StdStorage` to make this more robust? we've changed the ASR storage layout and deployed new ASR's a few times recently

Reply:
Looking at https://github.com/foundry-rs/forge-std/blob/master/src/StdStorage.sol, you'd probably have to get a little hacky within the lib to make it work since it relies on storage. Not a blocker, can track as a future improvement

Reply:
Tracked here https://github.com/ethereum-optimism/superchain-ops/issues/1064, resolving this comment.